index.js

```diff
@@ -79,7 +79,7 @@ var Sqlite3Driver = Base.extend({
       constraint.push('UNIQUE');
     }
 
-    if (spec.defaultValue) {
+    if (spec.defaultValue !== undefined) {
```

Comment:
I'm thinking outside of falsey values. There's going to be a case where someone wants to set the `defaultValue` to an empty string.